docs/superpowers/plans/2026-05-07-semver-dev-builds.md

```
@@ -0,0 +1,581 @@
+- [ ] **Step 3: Sanity check from current main**
+
+Run: `./scripts/compute-dev-version.sh`
+Expected: a string like `0.8.1-dev.<N>` where `<N>` is `git rev-list v0.8.0..HEAD --count`. Confirm `<N>` matches the count manually.
+
+- [ ] **Step 4: Verify failure modes**
+
+```bash
+# In a tmp dir with no tags, should fail loudly
+tmp=$(mktemp -d) && (cd "$tmp" && git init -q && git commit --allow-empty -m init -q && \
+  /home/jack/GitHub/bifrost/scripts/compute-dev-version.sh; echo "exit=$?")
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)